hoomd/data/param_dict.py

```diff
@@ -0,0 +1,144 @@
+from copy import deepcopy
```

Comment:
I'd use a full name for the module like `parameter_dict.py` that matches the class name.